worlds/sa2b/Options.py

```diff
@@ -227,7 +227,8 @@ class Omosanity(Toggle):
 
 class Animalsanity(Toggle):
     """
-    Determines whether picking up counted small animals grants checks
+    Determines whether unique counted animals grants checks.
```

Comment:
I don't really know what "unique counted" is meant to mean here. Maybe just "unique" or "uniquely named"?

Reply:
So can it not just be "Determines whether animals grant checks"?

Reply:
You could use "unique counts/totals/numbers of animals grant checks"

Reply:
I would be fine with "Determines whether unique counts of animals grants checks."